### PR TITLE
Dynamic block refactor

### PR DESCRIPTION
**Pull request by brown-a2**
* Refactor Hero Block so that it uses the WP dynamic block method of outputting block code to the frontend rather then via the default standard saving to the DB.
* Add a new file called index.php in the block. This holds all the new frontend code.
* Refactored the index.js slightly to accommodate the dynamic block requirements and standardize it.
* Add in the PHP linter Git Action tool.

### CODE REVIEW

File: build/style-gutenburg.css [hunk 1]
```diff
@@ -6493,8 +6489,7 @@ Block: Staggered Box (Frontend styles)
   .edit-post-visual-editor .mojblocks-staggered-box__image,
   .block-editor-block-styles .mojblocks-staggered-box__image,
   .block-editor-inserter__menu-help-panel .mojblocks-staggered-box__image {
-    -webkit-filter: invert(100%);
-            filter: invert(100%);
+    filter: invert(100%);
```
Comment (BNewing) [inline build/style-gutenburg.css:6492]: Looks good so far :) how come some of the -webkit-filter things have been removed?

File: src/hero/index.js [hunk 2]
```diff
@@ -13,83 +12,83 @@ registerBlockType("mojblocks/hero", {
             type: 'string'
         },
         heroTitle: {
-            type: 'string',
-            source: 'html',
-            selector: 'h2'
+            type: 'string'
```
Comment (BNewing) [inline src/hero/index.js:15]: How come the other attributes come out?

**Review by BNewing — COMMENTED** [review]
Left some questions :D

File: README.md [hunk 3]
```diff
@@ -1,16 +1,41 @@
...
+* Accordion
+* Card
+* CTA
+* Hero
+* Highlights-list
+* Intro
+* Quote
+* Reveal
+* Staggered-boxes
+* Video
```
Comment (RobLoweMOJ) [inline README.md:15]: Maybe worth a note saying Video is a placeholder..theres no block

**Review by RobLoweMOJ — APPROVED** [review]
👍

File: build/style-gutenburg.css (hunk 1 above)
Comment (brown-a2) [reply build/style-gutenburg.css:6492]: No, idea, this file gets compiled by Webpack. For this commit I've not done any CSS changes so it's funny something has changed there. I'll take a look and see if I can see anything that might have changed the compile.

File: src/hero/index.js (hunk 2 above)
Comment (brown-a2) [reply src/hero/index.js:15]: These are only used when you're building blocks the "standard" block way where you manipulate the saved HTML into the database. Now that we are using PHP to do this, these properties have no use (with the exception of type which PHP needs).

File: README.md (hunk 3 above)
Comment (brown-a2) [reply README.md:15]: Ah, okay I didn't realise video was a placeholder, I might remove completely then?

Comment (brown-a2) [reply README.md]: Removed video block for now

File: build/style-gutenburg.css (hunk 1 above)
Comment (brown-a2) [reply build/style-gutenburg.css]: I wonder if someone removed this but then didn't compile the CSS until I've come along and done it? Anyway, I've added it back in so hopefully that's what we want. It didn't seem to cause issue on my local machine.

**Pull request merged**